articles/digital-twins/concepts-models.md

```diff
@@ -166,7 +166,7 @@ Sometimes, you might want to define a relationship without a specific target, so
 
 Here's an example of a relationship on a DTDL model that doesn't have a target. In this example, the relationship is for defining what sensors a Room might have, and the relationship can connect to any type.
 
-:::code language="json" source="~/digital-twins-docs-samples-getting-started/models/advanced-home-example/IRoom.json" range="2-27" highlight="20-25":::
+:::code language="json" source="~/digital-twins-docs-samples-getting-started/models/advanced-home-example/IRoom.json" range="2-30" highlight="23-28":::
```

Comment:
Requested change: `highlight="22-27"`

Explanation: The highlight numbers actually start counting from the first line displayed in the doc snippet, not from the start of the file—so since the included range of this snippet starts at 2, the highlight count actually also starts at line 2, shifting the highlighted lines to be 22-27. I know that this is not exactly intuitive.
![image](https://github.com/user-attachments/assets/2ff070b7-ecc3-456f-9699-944f5b86c0b1)